### ISSUE TITLE
[BUG][v1.3.1] plugins.txt gets cut off sometimes because not all files start with *

### ISSUE THREAD

**Issue by phinocio** [phinocio/loadorderlibrary#34]
**Version Reported:** v1.3.1

**Initially Reported By:** Gatonegro

**Steps To Reproduce:**
- Upload a plugins.txt file that does not contain `*` at the beginning of each line

![example](https://cdn.discordapp.com/attachments/577715234752430082/815786126811201606/unknown.png)

**Comment by phinocio** [comment]
This is due to me simply doing 

```php
if ($fileName == "plugins.txt") {
	$line = substr($line, 1);
}
```
to remove the `*` from the beginning of the line due to my test file having it for every line. 

Likely solution is to do a check to see if the line starts with `*` and remove it only if it does (or simply do a `str_replace` for it).

**Comment by phinocio** [comment]
Fixed with v1.3.2
